### PR TITLE
Science beret sprites

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Resprites the science beret, making it more white than purple
Adds 2 new robotics berets

## Why It's Good For The Game
Requested by ramon for the colour swap on the science beret, regarding me making a janitorial one be majorly purple instead.
Robotics berets for more uniqueness between roles.

## Images of changes
image
image

## Testing
Spawn as RD, check loadout works, go to robodrobe, check beret is there.

## Changelog
:cl:
add: Two new robotics berets, in loadout and robodrobe
imageadd: New science beret sprite
/:cl:

### CODE REVIEW

**Review by Burzah — APPROVED** [review]

**Comment by Henri215** [comment]
Looks cool

File: code/modules/clothing/head/job_hats.dm [hunk 1]
```diff
@@ -215,8 +215,18 @@
 
 /obj/item/clothing/head/beret/sci
 	name = "science beret"
-	desc = "A purple beret with the science insignia emblazoned on it. It has that authentic burning plasma smell."
-	icon_state = "beret_sci"
+	desc = "A white beret with a purple science insignia emblazoned on it. It has that authentic burning plasma smell."
```
Comment (JimKil3) [inline code/modules/clothing/head/job_hats.dm:218]:
```suggestion
	desc = "A white beret with a purple science insignia emblazoned on it. It has that authentic smell of burning plasma."
```
minor nitpick

**Review by JimKil3 — APPROVED** [review]
LGTM :)

**Review by Henri215 — APPROVED** [review]
Sprites are looking good, could maybe rename to "beret_robotics_black/white", not that important

**Review by S34NW — APPROVED** [review]

**Pull request merged**